sources/__help.sh

```diff
@@ -0,0 +1,51 @@
+################
+#     HELP     #
+################
+
+function __help { 
+        cat << EOF
+Usage: app-deploy [OPTION] [ARGUMENTS]
+
+A group of scripts and tools designed to assist in the deployment process. This script can:
+  - Initialize deploy options.
+  - Create trigger tag for CI/CD.
+  - Extract environments from a trigger tag.
+  - Generate a final build tag after CI/CD deployment.
```

Comment:
Funnily enough I was writing a comment with a similar summary for the initial part of the readme. Maybe this just deserves to be in the readme intro, I think I prefer this outline to the big paragraph explaining all the details. The details / example workflow can come later on in the readme description.